### ISSUE TITLE
Compile error: bad symbolic reference. A signature in TypecheckerContextExtensions.class refers to term tools

### ISSUE THREAD

**Issue by aishfenton** [milessabin/macro-compat#59]
You also need to add scala-compiler as a dep. Worth mentioning in readme? 

```
"org.scala-lang" % "scala-compiler" % scalaVersion.value % "provided",
```

**Comment by milessabin** [comment]
Yes, definitely. PR? :-)

**Comment by aishfenton** [comment]
Done #60

**Comment by milessabin** [comment]
Fixed in #60. Many thanks :-)
